AppController/djinn.rb

```diff
@@ -4352,8 +4283,8 @@ def set_appcontroller_monit
     Djinn.log_debug("Configuring AppController monit.")
     service = `which service`.chomp
     start_cmd = "#{service} appscale-controller start"
-    stop_cmd = "#{service} appscale-controller stop"
     pidfile = '/var/run/appscale/controller.pid'
+    stop_cmd = "/sbin/start-stop-daemon --stop --pidfile #{pidfile} --retry=TERM/30/KILL/5"
```

Comment:
Should the path be resolved first with `which` like we do for other services, or do we have a guarantee that it will always be in /sbin?

Reply:
for now yes: we already have another hadcoded close by (service) I would say that we can revisit once we switch to systemd?